webrender/src/render_task.rs

```diff
@@ -701,7 +701,13 @@ impl RenderTask {
             }
         };
 
-        let (target_rect, target_index) = self.get_target_rect();
+        let (mut target_rect, target_index) = self.get_target_rect();
+        // The primitives inside a fixed-location render task
+        // are already placed to their corresponding positions,
+        // so the shader doesn't need to shift by the origin.
+        if let RenderTaskLocation::Fixed(_) = self.location {
+            target_rect.origin = DeviceIntPoint::origin();
```

Comment:
It seems like it might be tidier to ensure the target rect origin is initialized to zero for fixed tasks, instead of modified here? Not a bit deal either way.

Reply:
Unfortunately, we can't do this. There are other places where `Fixed(rect)` is expected to have the actual origin, e.g. `get_target_rect`.